### PR TITLE
Awin affiliate tracking

### PR DESCRIPTION
**Pull request by rupertbates**
This is a placeholder until we get the go ahead to continue with this.

### CODE REVIEW

File: app/views/main.scala.html [hunk 1]
```diff
@@ -46,5 +46,7 @@
 
         @fragments.global.footer(product, edition, contactUsCountry)
         @fragments.footerJavaScript()
+        <!--AWIN affiliate tracking tag - see here: https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW -->
+         <script src="https://www.dwin1.com/11152.js" type="text/javascript" defer="defer"></script>
```
Comment (johnduffell) [inline app/views/main.scala.html]: should this only be referenced on the thank you page when the variables are set?

**Comment by johnduffell** [comment]
looks good, the only issue is that the JS lib appears to be loaded and parsed on every page rather than just on checkout, I wonder if that would be bad for the page weight. 👍

**Comment by rupertbates** [comment]
Just for posterity and in answer to John's question about the lib being loaded on every page, this is according to the specification given by Awin. Docs are [here](https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW)

**Comment by johnduffell** [comment]
glad this one's finally out of limbo!! 🥇 well done
image

**Pull request merged**